### PR TITLE
Chore: Translation: Update zh-rTW strings

### PR DESCRIPTION
**Pull request by bluehomewu**
The words usage of the translation strings in the past are in the style of China rather than Taiwan.
I'm Taiwanese, who works on Android and Translation projects.

Signed-off-by: Edward Wu <bluehome.wu@gmail.com>

### CODE REVIEW

**Comment by bluehomewu** [comment]
CC: @Kaitul , @Kr328

**Comment by Kaitul** [comment]
@bluehomewu See this update: https://github.com/Kr328/ClashForAndroid/pull/1460

**Comment by bluehomewu** [comment]
> @bluehomewu See this update: #1460

I've checked it, but I still think there's a problem.

**Comment by Kaitul** [comment]
> > @bluehomewu See this update: #1460
> 
> I've checked it, but I still think there's a problem.

I think there are also some problems with your translation, and I think there should be a glossary that we can agree on.

**Comment by bluehomewu** [comment]
> 

Please point them.

**Comment by bluehomewu** [comment]
> > > @bluehomewu See this update: #1460
> > 
> > 
> > I've checked it, but I still think there's a problem.
> 
> I think there are also some problems with your translation, and I think there should be a glossary that we can agree on.

Please point them.

File: design/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -195,22 +195,25 @@
...
-    <string name="application_broken_tips">應用程式缺少必要的執行元件，這通常是由於下載了不完整的 Apk 而導致。</string>
+    <string name="application_broken_tips">應用程式缺少必要的執行元件，這通常是由於下載了不完整的 APK 而導致。</string>
     <string name="reinstall">重新安裝</string>
-    <string name="github_releases">Github Releases</string>
+    <string name="github_releases">GitHub Releases</string>
     <string name="unable_to_start_vpn">無法啟動 VPN 元件</string>
-    <string name="request_donate_tips">如果您覺得本應用對您有幫助歡迎在 [協助] 中給予開發人員一些捐助</string>
-    <string name="request_donate">捐助</string>
-    <string name="version_updated">應用已更新</string>
+    <string name="request_donate_tips">如果您覺得本應用程式對您有幫助歡迎在 [幫助] 中給予開發人員一些抖內</string>
```
Comment (pan93412) [inline design/src/main/res/values-zh-rTW/strings.xml:204]: 如果您覺得本應用程式對您有幫助「，」歡迎在 [幫助] 中給予開發人員一些抖內

加個逗號會比較好讀～

File: design/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
     <string name="edit">編輯</string>
     <string name="empty_name">空白名稱</string>
     <string name="exit_without_save">退出而不儲存</string>
-    <string name="exit_without_save_warning">所有變更將會丟失</string>
+    <string name="exit_without_save_warning">所有變更將會遺失</string>
```
Comment (pan93412) [inline design/src/main/res/values-zh-rTW/strings.xml:32]: 「將遺失所有變更」呢？不過原本的語句並沒什麼大問題。

**Comment by Kr328** [comment]
Is this PR ready for merging?

**Comment by Kaitul** [comment]
There are also some strings that need to be changed. @bluehomewu

**Comment by bluehomewu** [comment]
> Is this PR ready for merging?

Will do this on this weekend.

**Comment by bluehomewu** [comment]
@Kr328 Done. Finished works. Preparing for merge.

**Comment by Kr328** [comment]
thanks for your contribution

**Comment by Yuzurisa** [comment]
What exactly is 抖内 in Taiwannese?
Is it a kind of bad imitation of English culture?

**Comment by MonokuroInzanaito** [comment]
Yeah maybe it's due to Pelosi's visit.
Ultimately, Taiwan is a province of China, rather than an individual country. I wish her plane could have been shot down.

**Comment by bluehomewu** [comment]
> What exactly is 抖内 in Taiwannese? Is it a kind of bad imitation of English culture?

That's just a personalized translation, a playful word. To be close to the user's life.

**Comment by bluehomewu** [comment]
> Yeah maybe it's due to Pelosi's visit. Ultimately, Taiwan is a province of China, rather than an individual country. I wish her plane could have been shot down.

Here's GitHub, which talking about OpenSources and code, please don't talk about politics.

**Pull request merged**